docker/Dockerfile-api

```diff
@@ -15,9 +15,12 @@ RUN dnf -y install \
     openssl-devel \
     python3-devel \
     python3-pip \
+    python3-wheel \
+    && dnf update -y \
     && dnf clean all
 COPY . .
 COPY ./docker/iib-httpd.conf /etc/httpd/conf/httpd.conf
+RUN pip3 install --upgrade pip
```

Comment:
Could you please add a comment here explaining the issue

Reply:
added the comment